docs/modules/ROOT/pages/includes/attributes.adoc

```diff
@@ -0,0 +1,7 @@
+:quarkus-version: 2.12.1.Final
+:quarkus-github-app-version: 1.12.0
+
+:github-api-javadoc-root-url: https://github-api.kohsuke.org/apidocs/org/kohsuke/github
+:github-reference-documentation-root-url: https://docs.github.com/en/free-pro-team@latest/developers
+
+:command-airline-examples-dir: ./../examples/command/airline/
```

Comment:
@yrodiere I don't know what happened but your previous PR removed this file entirely. I noticed it and then forgot about it.

Reply:
Ah... It got updated by the build, I tried to revert that and I probably deleted it by mistake. Sorry...